packages/wdio-types/src/Capabilities.ts

```diff
@@ -1227,6 +1229,12 @@ export interface BrowserStackCapabilities {
      * @private
      */
     wdioService?: string
+    /**
+     * Specify an identifier for a build consists group of tests.
+     */
+    buildIdentifier?: string
+    'browserstack.buildIdentifier'?: string
+    'browserstack.localIdentifier'?: string
```

Comment:
Why should it be valid to define these caps:

```js
capabilities: [{
    browserName: 'chrome',
    // ...
    'bstack:options': {
        'browserstack.buildIdentifier': '123'
    }
}]
```

Are your customers already use it this way?